### PR TITLE
decode should stop when the truc is implementing TextUnmarshaller even if there is no env associated

### PR DESCRIPTION
**Pull request by Nexucis**

### CODE REVIEW

File: decode.go [hunk 1]
```diff
@@ -46,16 +46,16 @@ func (l *Lamenv) decode(conf reflect.Value, parts []string) error {
...
 			delete(l.env, variable)
 			if err := p.UnmarshalText([]byte(input)); err != nil {
 				return err
 			}
 
 			// in case the method UnmarshalEnv() is setting some parameter in the struct, we have to save these changes
 			v.Set(ptr.Elem())
-			return nil
 		}
+		return nil
```
Comment (celian-garcia) [inline decode.go:58]: Isn't there the same problem in UnmarshalEnv section?

Comment (Nexucis) [reply decode.go:58]: actually no, if you look closer, the `return nil` is at the correct place. So should be good

Comment (celian-garcia) [reply decode.go:58]: nevermind

Comment (celian-garcia) [reply decode.go:58]: Yeah I checked after, I thought it was a copy paste :P

**Pull request merged**